src/pubtools/_impl/tracing.py

```diff
@@ -60,7 +69,16 @@ def _reset(self):
         # trace.set_tracer_provider global singleton set below can *never* be set up
         # more than once in a single process.
 
-        self._enabled_trace = os.getenv("OTEL_TRACING", "").lower() == "true"
+        self._enabled_trace = (
+            os.getenv("OTEL_TRACING", "").lower() == "true"
+        ) and OPENTELEMETRY_AVAILABLE
+        if (
+            os.getenv("OTEL_TRACING", "").lower() == "true"
+        ) and not OPENTELEMETRY_AVAILABLE:
+            log.debug(
```

Comment:
Ideally there should be a test which reaches this line.

You could probably `monkeypatch.setattr(tracing, "OPENTELEMETRY_AVAILABLE", False)` from within a test to force it to behave as though otel libraries are not available even when they are.